src/traces/contourgl/convert.js

```diff
@@ -32,7 +32,7 @@ function Contour(scene, uid) {
     this.bounds = [0, 0, 0, 0];
 
     this.contourOptions = {
-        z: new Float32Array(),
+        z: new Float32Array(0),
```

Comment:
Interesting that this is required (I assume the missing `0` caused issue somewhere?)